content/docs/guides/grafbase.md

```diff
@@ -0,0 +1,199 @@
+---
+title: Use Grafbase resolvers with Neon
+subtitle: Learn how to build and deploy serverless GraphQL backend with Grafbase and Neon
+enableTableOfContents: true
+isDraft: true
+---
+
+_This guide was contributed by Josep Vidal from Grafbase_
+
+Grafbase allows you to combine your data sources into a centralized GraphQL endpoint and deploy a serverless GraphQL backend.
+
+This guide describes how to create a GraphQL API using Grafbase and use Grafbase [resolvers](https://website-git-gb-3006-add-changelog-for-resolvers.grafbase-vercel.dev/docs/edge-gateway/resolvers) with the [Neon serverless driver](/docs/serverless/serverless-driver) to interact with your Neon database at the edge.
+
+The example project in this guide simulates a marketplace of products, where the product price is dynamically calculated based on data retrieved from your Neon database.
+
+## Prerequisites
+
+- The [Grafbase CLI](https://website-git-gb-3006-add-changelog-for-resolvers.grafbase-vercel.dev/cli)
+- A Neon project. See [Create a Neon project](https://neon.tech/docs/manage/projects#create-a-project).
+
+## Create a backend with Grafbase
+
+1. Create a directory and initialize your Grafbase project by running the following commands:
+
+    ```bash
+    mkdir grafbase-neon
+    cd grafbase-neon
+    npx grafbase init
+    ```
+
+2. In your project directory, open the `grafbase/schema.graphql` file and replace the existing content with the following schema:
+
+    ```graphql
+    extend type Mutation {
+      addProductVisit(productId: ID!): ID! @resolver(name: "add-product-visit")
+    }
+
+    type Product @model {
+      name: String!
+      price: Float @resolver(name: "product/price")
+    }
+    ```
+
+## Create the schema in Neon
+
+1. Navigate to the Neon Console and select your project.
+2. Open the Neon **SQL Editor** and run the following `CREATE TABLE` statement:
+
+    ```sql
+    CREATE TABLE product_visits(id SERIAL PRIMARY KEY, product_id TEXT NOT NULL);
+    ```
+
+    The `product_visits` table stores product page view data that the application uses to dynamically calculate a product price.
+
+## Create the resolvers
+
+The schema includes an `addProductVisit` query and `prodcut/price` field. Create resolvers for those by creating the following files in your project directory:
+
+- `grafbase/resolvers/add-product-visit.js`
+- `grafbase/resolvers/product/price.js`
+
+You can use the following command to create the files:
+
+```bash
+cd grafbase
+mkdir resolvers
+cd resolvers
+touch add-product-visit.js
+mkdir product 
+touch price.js
+```
+
+Add the following code to each file:
+
+  ```graphql
+  export default function Resolver(root, params) {
+    // ...
+  }
+  ```
+
+You will expand the code in these files in a later step.
+
+## Install the Neon serverless driver
+
+Inside the `grafbase` directory in your project, run the following commands to install the Neon serverless driver:
+
+  ```bash
+  cd grafbase-neon/grafbase
```

Comment:
```suggestion
cd ..
```
After following the previous instructions, I was in the /resolvers folder. So the `cd grafbase-neon/grafbase` returns an error